File: app/src/main/java/maxzonov/vkapi_sandbox/utils/DateFormatter.java
package maxzonov.vkapi_sandbox.utils;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.TimeZone;

public class DateFormatter {

    public static String convertDateToDayString(long date) {
        Date defaultDate = new Date(date * 1000L);

        DateFormat dateFormat = new SimpleDateFormat("dd MMMM yyyy");

        return dateFormat.format(defaultDate);
    }

    public static String convertDateToTimeString(long date) {
        Date defaultDate = new Date(date * 1000L);

        DateFormat dateFormat = new SimpleDateFormat("hh:mm");
        dateFormat.setTimeZone(TimeZone.getTimeZone("GMT+3"));

        return dateFormat.format(defaultDate);
    }
}

File: app/src/main/java/maxzonov/vkapi_sandbox/utils/DateFormatter.kt
package maxzonov.vkapi_sandbox.utils

import java.text.SimpleDateFormat
import java.util.Date
import java.util.TimeZone

object DateFormatter {

    fun convertDateToDayString(date: Long): String {
        val defaultDate = Date(date * 1000L)

        val dateFormat = SimpleDateFormat("dd MMMM yyyy")

        return dateFormat.format(defaultDate)
    }

    fun convertDateToTimeString(date: Long): String {
        val defaultDate = Date(date * 1000L)

        val dateFormat = SimpleDateFormat("HH:mm")
        dateFormat.timeZone = TimeZone.getTimeZone("GMT+3")

        return dateFormat.format(defaultDate)
    }
}
